### DISCUSSION TITLE
Internal links in markdown not working?

### DISCUSSION THREAD

**Discussion by dealmeidavf** [jupyter/notebook#7418] [Q&A]
My internal links do not work anymore. Initially I thought was the move to Notebook 7 but they do not work on Notebook 6.5.7 either. Also they do not work on `nbclassic`.
I use firefox with nb 7 (on Gentoo Linux) and safari with nb 6.5.7 (on macOS Ventura).

On a markdown cell I do:

> `[Go to Power system block diagrams](#diag)`

Then later on the notebook on another cell after entering many blank cells so to get the first cell out of sight:

> `Power System Block Diagrams<a id=“diag”></a>`

Clicking on the link above *Go to Power system block diagrams* does not move the cursor down to the *Power System Block Diagrams* section.

Thanks for inputs.

Valmor

PS: I also tried on Binder: [Jupyter Notebook 7.1 on Binder](https://gist.github.com/jtpio/83b0b625cb3b6e05d7f3d8714f98182e) and no luck. If anyone has a work around this besides using the table of contents in Notebook 7, please let me know. At the moment I can't use Notebook 7 because on all my systems the `\newcommand` LaTeX macro [does not work](https://github.com/jupyter/notebook/discussions/7635) which makes my notebooks useless to edit the math writing. At the moment I am stuck with `nbclassic`.

**Comment by khanahmedm** [comment]
I am experiencing the same. I have jupyter 7.2.2.

**Comment by n-traore** [comment]
Same for me with notebook 7.2.2 on JupyterHub.
However when I open the notebook locally using vscode it works properly. Maybe an issue with the Jupyter server ?

**Comment by clemwangds** [comment]
I dusted off my old notebook from 7 years ago(!) and my internal links no longer work in Jupyter Lab Version 4.4.3

I have in my first cell:

`### <a id="toc">Table of Contents</a>`

And sprinkled throughout my other text cells:
`[toc](#toc)`
